packages/components/inline-adder/README.md

```diff
@@ -23,13 +23,14 @@ angular.module('myModule', ['oui.inline-adder'])
 
 ## Component `oui-inline-adder`
 
-| Attribute        | Type      | Binding   | One-time binding  | Values            | Default   | Description
-| ----             | ----      | ----      | ----              | ----              | ----      | ----
-| `on-add`         | function  | &         | no                | n/a               | n/a       | handler triggered when a row is added
-| `on-remove`      | function  | &         | no                | n/a               | n/a       | handler triggered when a row is removed
-| `on-change`      | function  | &         | no                | n/a               | n/a       | handler triggered when rows have changed
-| `tooltip-add`    | string    | @?        | yes               | n/a               | n/a       | tooltip to display for the add button
-| `tooltip-delete` | string    | @?        | yes               | n/a               | n/a       | tooltip to display for the delete button
+| Attribute        | Type      | Binding   | One-time binding  | Values            | Default          | Description
+| ----             | ----      | ----      | ----              | ----              | ----             | ----
+| `on-add`         | function  | &         | no                | n/a               | n/a              | handler triggered when a row is added
+| `on-remove`      | function  | &         | no                | n/a               | n/a              | handler triggered when a row is removed
+| `on-change`      | function  | &         | no                | n/a               | n/a              | handler triggered when rows have changed
+| `tooltip-add`    | string    | @?        | yes               | n/a               | n/a              | tooltip to display for the add button
+| `tooltip-delete` | string    | @?        | yes               | n/a               | n/a              | tooltip to display for the delete button
+| `rows-limit`     | integer   | <?        | yes               | n/a               | 9007199254740991 | maximum number of oui-inline-adder-rows
```

Comment:
**nitpick:** For better readability you can mention `Number.MAX_SAFE_INTEGER` instead of some random number.
```suggestion
| `rows-limit`     | integer   | <?        | yes               | n/a               | Number.MAX_SAFE_INTEGER | maximum number of oui-inline-adder-rows
```